src/main.js

```diff
@@ -262,18 +250,22 @@ function clearTimerSection() {
   removeClassProperty(displayUserTimer, 'hidden');
 }
 
-// EASY TODO: could make it a for loop
 function clearCatButtonSection() {
-buttonText[0].parentNode.classList.remove('study-active');
-buttonText[1].parentNode.classList.remove('meditate-active');
-buttonText[2].parentNode.classList.remove('exercise-active');
+  var buttonNames = ['study-active', 'meditate-active', 'exercise-active']
+  for (var i = 0; i < catButtons.length; i++) {
+    removeClassProperty(catButtons[i], buttonNames[i]);
+  }
 }
 
 // TODO: could do a queryALL to refactor
 function clearUserInputsSection() {
-  accomplishInput.value = '';
-  minutesInput.value = '';
-  secondsInput.value = '';
+  var inputs = document.querySelectorAll('input');
+  for (var i = 0; i < inputs.length; i++) {
+    inputs[i].value = '';
+  }
+  // accomplishInput.value = '';
+  // minutesInput.value = '';
+  // secondsInput.value = '';
 }
 
 function clearStartCircleColor() {
```

Comment:
We can combine this function into clearCatButtonSection, the code is almost the same